akka-docs/src/main/paradox/mailboxes.md

```diff
@@ -1,16 +1,8 @@
 # Classic Mailboxes
 
-@@@ note
-
-Akka Classic is the original Actor APIs, which have been improved by more type safe and guided Actor APIs, 
-known as Akka Typed. Akka Classic is still fully supported and existing applications can continue to use 
-the classic APIs. It is also possible to use Akka Typed together with classic actors within the same 
-ActorSystem, see @ref[coexistence](typed/coexisting.md). For new projects we recommend using the new Actor APIs.
-
+@@include[includes.md](includes.md) { #actor-api }
 For the new API see (FIXME https://github.com/akka/akka/issues/27124).
```

Comment:
I think that link can be added now typed/mailboxes.md

Reply:
added